src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -789,7 +789,7 @@ Status ServiceBasedNodeInfoAccessor::AsyncGetInternalConfig(
       [callback](const Status &status, const rpc::GetInternalConfigReply &reply) {
         std::unordered_map<std::string, std::string> result;
         if (status.ok() && reply.has_config()) {
-          result.insert(reply.config().config().begin(), reply.config().config().end());
+          result = MapFromProtobuf(reply.config().config());
         }
         if (!status.ok()) {
           RAY_LOG(ERROR) << "Failed to get internal config: " << status.message();
```

Comment:
local variable `result` is not needed with following code. and can you add some DEBUG log ?
```
if (status.ok() && reply.has_config()) {
  call_back(status, reply.config().config());
} else {
  call_back(status, boost::none);
  RAY_LOG(ERROR) << "Failed to get internal config: " << status.message();
}
```

Reply:
Done!

Reply:
I had to do something a bit different, but similar